src/pipeline/frames.rs

```diff
@@ -66,15 +67,16 @@ impl Iterator for FrameIterator {
         match self {
             FrameIterator::Image(ref mut img) => img.take(),
             FrameIterator::Video(ref mut video) => capture_video_frame(video),
-            FrameIterator::AnimatedGif {
+            FrameIterator::AnimatedImage {
                 ref frames,
                 ref mut current_frame,
             } => {
-                if *current_frame == frames.len() - 1 {
+                if *current_frame == frames.len() {
```

Comment:
caught this regression issue (my bad, sorry) when I tried a single frame webp

Reply:
not sure it was this line but well done, the previous version paused for a frame at the end of a gif . Reverting this change now skips ahead one frame which I think it's due to the subtracting one frame time when looping. Seems fine now